File: src/main/java/it/contrader/dto/OperatorDTO.java
package it.contrader.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor

public class OperatorDTO {


	private long id;
	
	private String name;
	
	private String type;
	
	private DepartmentDTO departmentDTO;
	
	private MachineDTO machineDTO;
	
	private ReportDTO reportDTO;

}

File: src/dto/operatordto.ts
import { DepartmentDTO } from './departmentdto';
import { MachineDTO } from './machinedto';

export class OperatorDTO {

	id: number;
	
	name : string;

	type : string;

	Department : DepartmentDTO;

	Machine: MachineDTO;
	
}